frontend/src/features/projects/spl/forms/CloseOutFinancialsForm.tsx

```diff
@@ -16,7 +15,8 @@ const CloseOutFinancialsForm = (props: CloseOutFinancialsFormProps) => {
       <Form.Row>
         <Col>
           <ProjectNotes
-            field={`notes[${NoteTypes.LoanTerms}].note`}
+            data-testid="loanTermsNote"
+            field="loanTermsNote"
             label="Loan Terms"
```

Comment:
Ideally, you should be able to get the note field via this label. If that isn't working we should try and fix the label.

Reply:
After your comment about how the testing library wants to work I instead looked for the label.  It solved the issue.